pkg/gitmanager/setup.go

```diff
@@ -47,7 +47,12 @@ func (gm *gitManager) Clone(ctx context.Context, payload *core.Payload, cloneTok
 		return err
 	}
 
-	if err = os.Rename(repoName+"-"+commitID, global.RepoDir); err != nil {
+	filename := repoName + "-" + commitID
+	if payload.GitProvider == core.Bitbucket {
+		filename = repoItems[len(repoItems)-2] + "-" + repoName + "-" + commitID[:12]
+	}
```

Comment:
this is cryptic. Can you write in comments an example value that'll be contained here? Also, won't it be better if we download the archive in a definite filename?